src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -757,6 +757,7 @@ public partial class UdpClient : System.IDisposable
         public byte[] Receive([System.Diagnostics.CodeAnalysis.NotNullAttribute] ref System.Net.IPEndPoint? remoteEP) { throw null; }
         public System.Threading.Tasks.Task<System.Net.Sockets.UdpReceiveResult> ReceiveAsync() { throw null; }
         public int Send(byte[] dgram, int bytes) { throw null; }
+        public int Send(ReadOnlySpan<byte> dgram) {throw null; }
```

Comment:
The param name as approved by API review is "datagram".

Reply:
Fixed, thanks.